### ISSUE TITLE
I try to write the training phase, but there are some problems occurring

### ISSUE THREAD

**Issue by Sweetclover** [saic-mdal/HiDT#10]

**Comment by belkakari** [comment]
You can ask about the problems here, we will be happy to help

**Comment by Sweetclover** [comment]
After computing the loss value, the oldest styles are removed
from the pool to keep its size at T.  

Can I choose 8 as the value of T?

**Comment by belkakari** [comment]
In the paper, we've used T=100. This was not an issue in terms of memory since we've stored only style vectors of relatively small size (3 for daytime and 12 for wikiart), but you definitely are welcome to experiment with the hyperparameters

**Comment by Sweetclover** [comment]
initial learning rate 0:0001 for both generators and discriminators, halving the learning rate every 200000 iterations.

But what's the total number of your iterations during training?

**Comment by denkorzh** [comment]
For the dataset of landscapes:
> We trained the translation model for 450 thousand iterations with batch size four on a single NVIDIA Tesla P40.
